src/main/java/org/benetech/servicenet/conflict/ConflictDetectionServiceImpl.java

```diff
@@ -58,17 +58,28 @@ public ConflictDetectionServiceImpl(ApplicationContext context,
     public void detect(List<OrganizationMatch> matches) {
         List<Conflict> conflicts = new LinkedList<>();
 
+        long detectionStartTime = System.currentTimeMillis();
         for (OrganizationMatch match : matches) {
-            log.debug("Request to detect conflicts for {} organization", match.getOrganizationRecord().getName());
+            long startTime = System.currentTimeMillis();
             OrganizationEquivalent orgEquivalent = organizationEquivalentsService.generateEquivalent(
                 match.getOrganizationRecord(), match.getPartnerVersion());
 
             List<EntityEquivalent> equivalents = gatherAllEquivalents(orgEquivalent);
 
             conflicts.addAll(detect(equivalents, match.getOrganizationRecord().getAccount(),
                 match.getPartnerVersion().getAccount()));
+            long stopTime = System.currentTimeMillis();
```

Comment:
please add TODO for removing this logic in the future, you cloud also add a link to an issue for refactoring the logs.